src/components/VideoCard/VideoCard.jsx

```diff
@@ -1,8 +1,13 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
 import * as styles from "./VideoCard.module.css";
 export function VideoCard() {
+  const navigate = useNavigate();
+  const handleVideo = () => {
```

Comment:
You can improve on namings here, instead of `handleVideo` you can change it to `navigateToWatch`